subt/main.py

```diff
@@ -904,24 +896,36 @@ def play_virtual_track(self):
         if self.use_right_wall == 'auto':
             self.use_right_wall = self.robot_name.endswith('R')
             self.use_center = self.robot_name.endswith('C')
-        self.stdout('Use right wall:', self.use_right_wall)
+        self.stdout('Use right wall:', self.use_right_wall)  # this can be now also misleading if we allow multiple changes
```

Comment:
If it can be misleading ... should we either print something else, or nothing?

Reply:
OK, I will add cleanup commit - this was kept to match old logfile 1:1 ... no longer needed.